plugins/SegmentEditor/API.php

```diff
@@ -405,20 +405,17 @@ public function getAll($idSite = false)
      * Filter out any segments which cannot be initialized due to disable plugins or features
      *
      * @param array $segments
-     * @param       $idSite
+     * @param bool|int $idSite
      *
      * @return array
      */
     private function filterSegmentsWithDisabledElements(array $segments, $idSite = false): array
```

Comment:
We could maybe try to get rid of such idSite parameters, that can either be a number or false. A better handling would imho be either a number or `null`. That would at some point even allow to type hint that parameter.
No need to change that now, but we should imho consider this for future improvements if we are already working on certain places of the code...